### ISSUE TITLE
ability to set Font/Size in DropList-objects

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roloww/gui4processing#18]
```
Many of the G4P objects allow you to change the font and size of the text 
display. This is not the case for the DropList-object.

the setFont-method is not defined nor inherited for the GDropList control.

eg. the GLabel-control inherits it from GTextAlign and the GTextField and 
GTextArea inherit it from GEditableTextControl.

Wouldn't it be more consistent to make setFont a method of GTextBase?


I am look for a way to change the size of the text displayed in a DropList.

John
```

Original issue reported on code.google.com by `famkemen...@gmail.com` on 12 Jul 2013 at 6:32

**Comment by GoogleCodeExporter** [comment]
```
Wouldn't it be more consistent to make setFont a method of GTextBase?

Yes it would - well spotted.

I am going to mark this one up as an enhancement since the GDropList does work.

```

Original comment by `p.k.la...@gmail.com` on 13 Jul 2013 at 1:31
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v3.2 available from [http://sourceforge.net/projects/g4p/ SourceForge]
```

Original comment by `p.k.la...@gmail.com` on 23 Jul 2013 at 9:41
- Changed state: **Fixed**
